static/images/how-it-works/how-it-works-2.svg

```diff
@@ -1 +1,30 @@
\ No newline at end of file
+<svg width="314" height="226" viewBox="0 0 314 226" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Nit: how-it-works-2.svg is sliiiightly different width/height than *-1.svg and *-3.svg:


```svg
<!-- static/images/how-it-works/how-it-works-1.svg -->
<svg width="306" height="223" viewBox="0 0 306 223" fill="none" xmlns="http://www.w3.org/2000/svg">
```

```svg
<!-- static/images/how-it-works/how-it-works-2.svg -->
<svg width="314" height="226" viewBox="0 0 314 226" fill="none" xmlns="http://www.w3.org/2000/svg">
```

```svg
<!-- static/images/how-it-works/how-it-works-3.svg -->
<svg width="306" height="223" viewBox="0 0 306 223" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Reply:
Ahhh, I thought those were standardized on Figma. I'll specify fixed width/heights, thanks @pdehaan